src/components/App.tsx

```diff
@@ -2741,6 +2743,86 @@ class App extends React.Component<AppProps, AppState> {
     }
   };
 
+  private handleEraser = (
+    event: PointerEvent,
+    pointerDownState: PointerDownState,
+    scenePointer: { x: number; y: number },
+  ) => {
+    const idsToUpdate: Array<string> = [];
+
+    const distance = distance2d(
+      pointerDownState.lastCoords.x,
+      pointerDownState.lastCoords.y,
+      scenePointer.x,
+      scenePointer.y,
+    );
+    const threshold = 10 / this.state.zoom.value;
+    const point = pointerDownState.lastCoords;
+    let samplingInterval = 0;
+    while (samplingInterval < distance) {
+      const hitElement = this.getElementAtPosition(point.x, point.y);
```

Comment:
let's use all elements under cursor (`getElementsAtPosition()`), not just the top-most. This will fix two major cases:

1) elements covering each other

![eraser_zindex](https://user-images.githubusercontent.com/5153846/157744134-6e4127ff-49f5-4e1b-bbb0-9133b2487ad7.gif)

2) elements (lines etc.) very close to each other but not covering each other, especially when moving the cursor fast so that there aren't many samples which to hitbox